### PR TITLE
nmctl: Adding command to get DHCP mode

### PR DESCRIPTION
**Pull request by ntsbtz**

### CODE REVIEW

**Comment by ssahani** [comment]
I do not get it . what is this PR for ?

File: src/manager/network-config-manager.c [hunk 1]
```diff
@@ -1549,7 +1549,7 @@ _public_ int ncm_link_replace_address(int argc, char *argv[]) {
         AddressFamily family = ADDRESS_FAMILY_NO;
         _auto_cleanup_strv_ char **addrs = NULL;
         _auto_cleanup_ IfNameIndex *p = NULL;
-        int r;
+        int r = -1;
```
Comment (ssahani) [inline src/manager/network-config-manager.c:1553]: Is this requred ?

File: src/manager/network-config-manager.c [hunk 2]
```diff
@@ -3385,7 +3385,7 @@ _public_ int ncm_link_remove_dhcpv4_server_static_address(int argc, char *argv[]
         _auto_cleanup_ char *mac = NULL;
         bool have_mac = false;
         bool have_address = false;
-        int r;
+        int r = -1;
```
Comment (ssahani) [inline src/manager/network-config-manager.c:3389]: Is this requred ?

File: src/manager/network-config-manager.c [hunk 3]
```diff
@@ -3454,7 +3454,7 @@ _public_ int ncm_link_add_ipv6_router_advertisement(int argc, char *argv[]) {
         IPv6RAPreference preference = _IPV6_RA_PREFERENCE_INVALID;
         _auto_cleanup_ IfNameIndex *p = NULL;
         _auto_cleanup_ char *domain = NULL;
-        int r;
+        int r = -1;
```
Comment (ssahani) [inline src/manager/network-config-manager.c:3457]: Is this requred ?

File: src/manager/network-config-manager.c (hunk 1 above)
Comment (ntsbtz) [reply src/manager/network-config-manager.c:1553]: yeah warning poped up.

**Pull request merged**